api/python/provisioner/__main__.py

```diff
@@ -132,6 +132,11 @@ def _set_logging(output_type, log_args=None, other_args=None):
         setattr(log_args, config.LOG_RSYSLOG_HANDLER, False)
         log_args.update_handlers()
 
+    if cmd in config.LOG_SUPPRESSION_CMDS:
+        setattr(log_args, "rsyslog_level", logging.WARNING)
+        setattr(log_args, "logfile_level", logging.WARNING)
```

Comment:
I believe we don not need that:
- logfile by default is not turned on for get command
- in case of custom logfile settings we should allow logs to custom path

Reply:
Left suppression only for **rsyslog** type of logging.